packages/app/src/client/components/withSession/index.tsx

```diff
@@ -438,12 +434,8 @@ async function authStatusRequestWithRetry(): Promise<
       withCredentials: true
     });
   } catch (err: any) {
-    if (!axios.isAxiosError(err)) {
-      // re-throw programming errors
-      throw err;
-    }
```

Comment:
This error was not handled anywhere. If this error is re-thrown like that, it will just result in a frozen UI with no feedback to the user.